android/build.gradle

```diff
@@ -32,7 +32,7 @@ buildscript {
             mavenCentral()
         }
         dependencies {
-            classpath 'com.android.tools.build:gradle:7.0.4'
+            classpath 'com.android.tools.build:gradle:7.0.2'
```

Comment:
any reason why we are downgrading 7.0.4 to 7.0.2?

Reply:
Not really, I thought it would fix the issues with javadoc but seems like it has nothing to do with it. Will update it back to 7.0.4